lxc/launch.go

```diff
@@ -96,7 +96,7 @@ func (c *launchCmd) run(config *lxd.Config, args []string) error {
 		}
 
 		if count != 2 {
-			return fmt.Errorf(gettext.Gettext("bad number of things scanned from resource"))
+			return fmt.Errorf(gettext.Gettext("bad number of things scanned from image, ccontainer or snapshot"))
```

Comment:
eh, I didn't originate that so I leave that for someone else to fix :)  ("things")  it does sound clunky.